src/transport/SessionManager.cpp

```diff
@@ -201,7 +201,28 @@ CHIP_ERROR SessionManager::PrepareMessage(const SessionHandle & sessionHandle, P
         packetHeader.SetSessionId(keyContext->GetKeyHash());
         CryptoContext::NonceStorage nonce;
         CryptoContext::BuildNonce(nonce, packetHeader.GetSecurityFlags(), packetHeader.GetMessageCounter(), sourceNodeId);
-        CHIP_ERROR err = SecureMessageCodec::Encrypt(CryptoContext(keyContext), nonce, payloadHeader, packetHeader, message);
+        CHIP_ERROR err = CHIP_NO_ERROR;
```

Comment:
This, I believe, is the GroupOutgoing session type in the switch-case. Group sessions would not support TCP. This logic should reside in the SecureSession case, isn't it? The SessionHandle->SessionType() is checked in the switch above.